js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/EvaluationDetailDialog.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogFooter,
+  DialogHeader,
+  Mono,
+  NonIdealState,
+  SpinnerWithText,
+} from '@dagster-io/ui-components';
+import {ReactNode, useState} from 'react';
+
+import {GET_EVALUATIONS_QUERY} from './GetEvaluationsQuery';
+import {PartitionTagSelector} from './PartitionTagSelector';
+import {QueryfulEvaluationDetailTable} from './QueryfulEvaluationDetailTable';
+import {GetEvaluationsQuery, GetEvaluationsQueryVariables} from './types/GetEvaluationsQuery.types';
+import {usePartitionsForAssetKey} from './usePartitionsForAssetKey';
+import {useQuery} from '../../apollo-client';
+import {DEFAULT_TIME_FORMAT} from '../../app/time/TimestampFormat';
+import {TimestampDisplay} from '../../schedules/TimestampDisplay';
+
+interface Props {
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+  assetKeyPath: string[];
+  evaluationID: string;
+}
+
+export const EvaluationDetailDialog = ({isOpen, setIsOpen, evaluationID, assetKeyPath}: Props) => {
+  return (
+    <Dialog isOpen={isOpen} onClose={() => setIsOpen(false)} style={EvaluationDetailDialogStyle}>
+      <EvaluationDetailDialogContents
+        evaluationID={evaluationID}
+        assetKeyPath={assetKeyPath}
+        setIsOpen={setIsOpen}
+      />
+    </Dialog>
+  );
+};
+
+interface ContentProps {
+  evaluationID: string;
+  assetKeyPath: string[];
+  setIsOpen: (isOpen: boolean) => void;
+}
+
+const EvaluationDetailDialogContents = ({evaluationID, assetKeyPath, setIsOpen}: ContentProps) => {
+  const [selectedPartition, setSelectedPartition] = useState<string | null>(null);
+
+  const {data, loading} = useQuery<GetEvaluationsQuery, GetEvaluationsQueryVariables>(
+    GET_EVALUATIONS_QUERY,
+    {
+      variables: {
+        assetKey: {path: assetKeyPath},
+        cursor: `${BigInt(evaluationID) + 1n}`,
+        limit: 2,
+      },
+    },
+  );
```

Comment:
Is this the intended way to retrieve an individual evaluation? It's what we're doing elsewhere, but it seems a little unusual.